src/constants.py

```diff
@@ -154,8 +154,24 @@
 DEFAULT_CONFIG = {
         "cooldown": 10,
         "port": 1100,
-        "weapon": "Vandal",
+        "weapons": "Vandal, Phantom",
         "chat_limit": 5,
+        "hide_skin_weapon_suffix": True,
+        "skin_abbreviations": {
+            "status": True,
+            "abbreviations": {
+                "Gravitational Uranium Neuroblaster": "GUN",
+                "Radiant Entertainment System": "RES",
+                "VALORANT GO! Vol. 1": "V. G. Vol. 1",
+                "VALORANT GO! Vol. 2": "V. G. Vol. 2",
+                "Radiant Crisis 001": "RC 001",
+                "Sentinels of Light": "SoL",
+                "Prelude to Chaos": "PtC",
+                "Winterwunderland": "Winter",
+                "Kohaku & Matsuba": "K&M",
+                "Gaia's Vengeance": "Gaia"
+            }
+        },
```

Comment:
Previously discussed: [here](https://github.com/zayKenyon/VALORANT-rank-yoinker/pull/161#issuecomment-1658976073).

Reply:
You can also add Tethered Realms, Nunca Olvidados, Radiant Crisis 001, Protocol 781-A, Divine Swine, Lycan's Bane, Coalition Cobra, Task Force 809, Piedra del Sol, Premiere Collision, Starlit Odyssey.
There are many long skin names making all of them short will make it confusing but we need to have a good system for this other wise there wont be space left in the table.

Reply:
> You can also add Tethered Realms, Nunca Olvidados, Radiant Crisis 001, Protocol 781-A, Divine Swine, Lycan's Bane, Coalition Cobra, Task Force 809, Piedra del Sol, Premiere Collision, Starlit Odyssey.

I added all of them, that are longer than 10 characters

> There are many long skin names making all of them short will make it confusing

I understand your point, but I would still appreciate suggestions on making it possible